public/app/plugins/panel/alertlist/module.tsx

```diff
@@ -148,6 +148,14 @@ const unifiedAlertList = new PanelPlugin<UnifiedAlertListOptions>(UnifiedAlertLi
       },
       category: ['Filter'],
     })
+    .addBooleanSwitch({
+      path: 'showInactiveAlerts',
+      name: 'Show alerts with 0 instances',
+      description:
+        'Show inactive alert rules (only for Prometheus alerts). If an Alert instance label filter is configured, inactive alert rules will be hidden',
```

Comment:
If there's more than one sentence in UI text, you have to include all periods. 

This UI text is a bit unclear. Do you mean that if you have the Alert instance label configured, it overrides this filter? If so, that should be said more explicitly and probably added in the docs.

```suggestion
        'Show inactive alert rules (for Prometheus alerts only). If the Alert instance label filter is configured, inactive alert rules will be hidden.',
```

Reply:
what do you mean by prometheus alert rules as opposed to other alert rule types?